sidecar/config/config.go

```diff
@@ -28,11 +28,17 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+const (
```

Comment:
How about: `on_exit` instead of `on_death`, and `restart`, `ignore`, and `terminate` as values?